pinc/ProjectSearchForm.inc

```diff
@@ -32,9 +33,9 @@ class ProjectSearchWidget
 
         if ( $this->type == 'text' )
         {
-            if ( isset($search_data[$this->id]) )
+            if ( isset($_GET[$this->id]) )
             {
-                $value_attr = "value='" . attr_safe($search_data[$this->id]) . "'";
+                $value_attr = "value='" . attr_safe($_GET[$this->id]) . "'";
```

Comment:
This file should `include_once($relPath."misc.inc");` for `attr_safe()` and now `array_get()` and not rely on something else bringing it in.